inst/validation/bbr-stories.yaml

```diff
@@ -220,6 +220,13 @@ MGMT-S016:
   - CLM-R002
   - CLM-R003
   - CLM-R004
+MGMT-S017:
+  name: Passing model file extensions
+  description: As a user, I want to be able to pass '.ctl' and '.mod' file extensions when making a new model.
```

Comment:
This seems to be tightening up the "ignore extensions" request from gh-486.  Given that request and that that is what the code does, is there a reason to make this story more specific than "ignores extensions"?